client/src/views/Room.vue

```diff
@@ -442,6 +443,18 @@ export default defineComponent({
 		onMounted(async () => {
 			await waitForToken(store);
 
+			if (history.state.created && store.state.settings.autoSkipSegmentCategories) {
+				try {
+					await API.patch(`/room/${route.params.roomId ?? store.state.room.name}`, {
+						autoSkipSegmentCategories: store.state.settings.autoSkipSegmentCategories,
+					});
+				} catch (e) {
+					console.error(e);
+				}
+
+				delete history.state.created;
+			}
+
```

Comment:
This feels very hacky. We can send the room settings directly to the `/api/room/create` and `/api/room/generate` endpoints, and avoid the extra round trip.